admin-dev/themes/new-theme/js/components/form-submit-button.js

```diff
@@ -60,11 +60,32 @@ export default class FormSubmitButton {
         return;
       }
 
+      let method = 'POST';
+      let addInput = null;
+
+      if ($btn.data('method')) {
+        const btnMethod = $btn.data('method');
+        const isGetOrPostMethod = ['GET', 'POST'].includes(btnMethod);
+        method = isGetOrPostMethod ? method : 'POST';
```

Comment:
It seems like the value of the method variable can only be 'POST'. Or is there something I don't get ?

Reply:
No it can be GET or POST, and for other methods (HEAD, DELETE) an hidden input is added and the http method used is POST

Reply:
yes but the **method** variable is set to 'POST' and never changed after...  I think he meant to use the value of **btnMethod** instead. 🤔

Reply:
If the method is something else than GET or POST, POST will be set as verb and the hidden input will do the job to make the request with the correct verb